cronjobs/fetch_marc_updates.py

```diff
@@ -129,8 +130,71 @@ def AddToCumulativeCollection(downloaded_file, config):
         shutil.copy(downloaded_file, output_directory)
     except Exception as e:
         util.Error("Adding file to cumulative collection failed! (" + str(e) + ")")
+
     return None;
 
+def CumulativeFilenameGenerator(output_directory):
+     return os.listdir(output_directory)
+
+# We try to keep all differential update up to the last complete data
```

Comment:
Maybe "We try to keep all differential updates up to and including the last complete data."?